### PR TITLE
PLF-6592: Update Upgrade plugin to match the new API behavior

### PR DESCRIPTION
**Pull request by boubaker**
Avoid to execute component plugins each Server Startup (After API improvement - PLF-6592).

### CODE REVIEW

File: component/upgrade/plugins/src/main/java/org/exoplatform/platform/upgrade/plugins/UpgradeProductInfoNodePlugin.java [hunk 1]
```diff
@@ -116,8 +117,8 @@ public void processUpgrade(String oldVersion, String newVersion) {
   @Override
   public boolean shouldProceedToUpgrade(String newVersion, String previousVersion) {
       // --- return true only for the first version of platform
-      //return VersionComparator.isAfter(newVersion,previousVersion);
-      return true;
+      return VersionComparator.isAfter(newVersion,previousVersion);
+      //return true;
```
Comment (thomasdelhomenie) [inline component/upgrade/plugins/src/main/java/org/exoplatform/platform/upgrade/plugins/UpgradeProductInfoNodePlugin.java]: Useless comment

File: component/upgrade/plugins/src/main/java/org/exoplatform/platform/upgrade/plugins/UpgradeProductInfoNodePlugin.java [hunk 2]
```diff
@@ -116,8 +117,8 @@ public void processUpgrade(String oldVersion, String newVersion) {
   @Override
   public boolean shouldProceedToUpgrade(String newVersion, String previousVersion) {
       // --- return true only for the first version of platform
```
Comment (thomasdelhomenie) [inline component/upgrade/plugins/src/main/java/org/exoplatform/platform/upgrade/plugins/UpgradeProductInfoNodePlugin.java]: Is this comment still valid ?

Comment (boubaker) [inline component/upgrade/plugins/src/main/java/org/exoplatform/platform/upgrade/plugins/UpgradeProductInfoNodePlugin.java]: Considered

File: component/upgrade/plugins/src/main/java/org/exoplatform/platform/upgrade/plugins/UpgradeProductInfoNodePlugin.java (hunk 1 above)
Comment (boubaker) [inline component/upgrade/plugins/src/main/java/org/exoplatform/platform/upgrade/plugins/UpgradeProductInfoNodePlugin.java]: Considered

**Pull request merged**